src/modules/organization/home/team/TeamPage.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div
+    class="w-full h-full bg-gray-1 flex flex-col py-4 items-center content-min-height"
+  >
+    <div class="flex flex-col content-max-width">
+      <OrgBanner :org="test.eventOrganizer" class="mb-3" />
+      <section class="flex justify-between mb-3">
+        <div class="flex items-center">
+          <input

```

Comment:
Search is very identical, can we reuse this as a component?
![image](https://user-images.githubusercontent.com/12471844/114165929-5019ba80-9957-11eb-8c1b-aeff177045b4.png)